ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.hbs

```diff
@@ -52,16 +52,16 @@
 {{/is}}
 {{#is type 'result'}}
     <div>
-        <h3 class="search-form-title" data-help="{{name}}">{{name}}</h3>
+        <h3 class="search-form-title" data-help="{{title}}">{{title}}</h3>
             <span class="search-form-contents">
                     {{createdOn}}
             </span>
         <span class="search-form-contents">
             <span class="fa fa-cloud"></span>
             {{createdBy}}
         </span>
-        <span class="choice-actions is-button" title="Shows a list of actions to take on the search forms" 
-        data-help="Shows a list of actions to take on the search forms.">
+        <span class="choice-actions is-button" title="Shows a list of actions to take on the result forms"
+        data-help="Shows a list of actions to take on the result forms.">
```

Comment:
why was this changed?  is this not the handle bars for search forms?

Reply:
Yes, but search and result forms share a lot of the same files, including this one. On line 53, there's a tag for `#is type result`, so the code within that tag is only used for result forms

Reply:
👍